BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -765,11 +786,14 @@ Unfortunately, many TLS/DTLS cipher suites were defined that do not feature forw
 
 For performance reasons, many TLS implementations reuse Diffie-Hellman and Elliptic Curve Diffie-Hellman exponents across multiple connections. Such reuse can result in major security issues:
 
-* If exponents are reused for too long (e.g., even more than a few hours), an attacker      who gains access to the host can decrypt previous connections. In other words, exponent reuse negates the effects of forward secrecy.
+* If exponents are reused for too long (e.g., even more than a few hours), an attacker who gains access to the host can decrypt previous connections. In other words, exponent reuse negates the effects of forward secrecy.
 
 * TLS implementations that reuse exponents should test the DH public key they receive for group membership, in order to avoid some known attacks. These tests are not standardized in TLS at the time of writing. See {{?RFC6989}} for recipient tests required of IKEv2 implementations that reuse DH exponents.
-      
-      
+
+* Under certain conditions, the use of static DH keys, or of ephemeral DH keys that are reused across multiple connections, can lead to timing attacks (such as those described in {{RACCOON}}) on the shared secrets used in Diffie-Hellman key exchange.
+
+To address these concerns, TLS implementations SHOULD NOT use static DH keys and SHOULD NOT reuse ephemeral DH keys across multiple connections.
```

Comment:
draft-bartle-tls-deprecate-ffdhe-00 has MUST NOT for static DH. We should be in sync with them. Maybe just add a "TODO: revisit when draft-bartle is published".

Reply:
MUST seems like the safest advice, although I suppose there are tradeoffs involved as always. I'll modify the draft as you suggest.